common/helpers/front_end.py

```diff
@@ -50,6 +50,22 @@ def get_page_path_parameters(url, page_section_generator=None):
     match = page_section_generator['regex'].search(url)
     return match.groupdict()
 
+def clean_invalid_args(url_args):
```

Comment:
A cleaner way to handle the logic in this method would be to
- Extract args dictionary
- Remove id/section from dictionary
- Reconstruct url from dictionary using args_dict_to_string

Reply:
Thanks for pointing out that function for me to use. I have incorporated it and write the code in a much cleaner way.